caddisfly-app/app/src/main/assets/tests.json

```diff
@@ -372,6 +375,63 @@
         }
       ]
     },
+    {
```

Comment:
Sorry for the formatting weirdness of my IDE, but the actual added test is right here